src/styles/blocks/button.scss

```diff
@@ -0,0 +1,21 @@
+.button {
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding